cli/default-reporter/test/reportingLifecycleScripts.ts

```diff
@@ -490,10 +511,11 @@ ${chalk.cyan('packages/foo')} ${POSTINSTALL}: Done`)
   })
 })
 
-test('groups lifecycle output when hideLifecyclePrefix is used', (done) => {
+test('groups lifecycle output when append-only and reporter-hide-prefix are used', async () => {
   const output$ = toOutput$({
     context: { argv: ['install'] },
     reportingOptions: {
+      appendOnly: true,
       outputMaxWidth: 79,
       hideLifecyclePrefix: true,
     },
```

Comment:
The `--reporter-hide-prefix` only works for `append-only`, so I changed this test